src/service/localization/index.js

```diff
@@ -35,21 +35,15 @@ function findLocalizedString(
   languageCodes,
   localizedStringId
 ) {
-  let localizedString = null;
-
-  languageCodes.some((languageCode) => {
-    const localizedStringBundle = localizedStringBundles[languageCode];
-    if (localizedStringBundle && localizedStringBundle[localizedStringId]) {
-      localizedString =
-        localizedStringBundle[localizedStringId].string ||
-        localizedStringBundle[localizedStringId].fallback;
```

Comment:
Are we dropping `fallback` support?

Reply:
Yes, I checked with Jon (original author of the extension) and the fallback value was never used (and there's no need to use it with the current fallback languages implementation). We agreed it's ok to remove to allow for this, as it's a feature that's not being used and we don't plan to use.

Reply:
Thanks for confirming!